src/helpers/getChainDetails.ts

```diff
@@ -0,0 +1,55 @@
+interface Chain {
+  rpc?: string;
+  name: string;
+  logoUri: string;
+}
+interface ChainInterface {
+  1: Chain;
+  137?: Chain;
+  56?: Chain;
+  42161?: Chain;
+  43114?: Chain;
+  250?: Chain;
+}
+
+const chainDetails: ChainInterface = {
+  1: {
+    rpc: process.env.REACT_APP_RPC_PROVIDER_ETH,
+    name: "Ethereum",
+    logoUri:
+      "https://storageapi.fleek.co/d1921602-c1d0-4d59-82e9-e36a2947b855-bucket/App/static/token-icons/weth.png",
+  },
+  137: {
+    rpc: process.env.REACT_APP_RPC_PROVIDER_POLYGON,
+    name: "Polygon",
+    logoUri: "https://wallet-asset.matic.network/img/tokens/matic.svg",
```

Comment:
We probably want to store all of this on IPFS ourselves.

Reply:
Yeah I agree for the logoURI's